src/orbit.rs

```diff
@@ -172,16 +172,10 @@ pub async fn load_orbit(oid: Cid, path: PathBuf) -> Result<Option<Orbit>> {
 // 1min timeout to evict orbits that might have been deleted
 #[cached(size = 100, time = 60, result = true)]
 async fn load_orbit_(oid: Cid, dir: PathBuf) -> Result<Orbit> {
-    let mut cfg = Config::new(Some(dir.join("block_store")), 0);
-    cfg.network.mdns = None;
```

Comment:
is mDNS needed for the gossiping?

Reply:
No, only for peer discovery on common networks